tests/test_types.py

```diff
@@ -76,3 +77,21 @@ def test_emit_policy_period():
     policy = _dict_to_emit_policy(policy_dict)
     assert type(policy) == EmitAfterPeriod
     assert policy.delay_in_seconds == 8
+
+
+@pytest.mark.parametrize(
+    "description",
+    [
+        # taosBenchmark
+        # list(conn.query("describe test.d6241;"))
+        ("ts", "TIMESTAMP", 8, "", "delta-i", "lz4", "medium"),
+        ("current", "FLOAT", 4, "", "delta-d", "lz4", "medium"),
+        ("voltage", "INT", 4, "", "simple8b", "lz4", "medium"),
+        ("phase", "FLOAT", 4, "", "delta-d", "lz4", "medium"),
+        ("groupid", "INT", 4, "TAG", "disabled", "disabled", "disabled"),
+        ("location", "VARCHAR", 24, "TAG", "disabled", "disabled", "disabled"),
+    ],
+)
+def test_description_parsing_to_tdengine_field(description: tuple) -> None:
+    """Test that the parsing works"""
```

Comment:
Only happy flows are tested here.
But for example - more/fewer fields than expected will result in an error.

Reply:
That can be said for any constructor or function that doesn't take `*args` though. It doesn't really count as parsing. And doesn't really justify a test if I'm not misunderstanding something here.

Reply:
This test is beneficial for the following reasons:
1. Increases the coverage in the non-integration tests that run in the CI. TDEngine integration tests are skipped.
2. Insures a regression in the class - e.g. removing adding a field - will be caught.

Reply:
Would be nice to turn it into a test of actual functionality by testing the logic in `_get_table_schema` (preferably by moving everything other than `self._connection.query(...)` to another method and testing that).

Then it can test error cases as well.